src/components/LicenseForm/components/index.js

```diff
@@ -1,2 +1 @@
-export { default as OrganizationsFieldArray } from './OrganizationsFieldArray';
-export { default as TermsListField } from './TermsListField';
+export { default } from './TermsListField';
```

Comment:
Let's not export it as default. It's counter-intuitive to be default exporting a particular component from a `components` directory. Just add an `ignore` for the eslint rule that complains that named exporting without a default export.